makefiles/boards/alias.inc.mk

```diff
@@ -0,0 +1,10 @@
+# list of known alias
+ALIAS.phynode   = pba-d-01-kw2x
+#TODO remove this line (as its just for show)
+ALIAS.linux     = native
+
+# this temporary variable is needed as we can not use BOARD when assigning BOARD
+ALIAS_TARGET := $(ALIAS.$(BOARD))
+ifdef ALIAS_TARGET
```

Comment:
Better to check for non-empty value: `ifneq (,$(ALIAS_TARGET))`

Reply:
actually, this did for some reason not work. Not sure if I had a typo or if something else went wrong. Could you maybe quickly verify if the non-empty value check works for you?